hazelcast/src/main/java/com/hazelcast/spi/impl/eventservice/impl/EventServiceImpl.java

```diff
@@ -677,18 +676,9 @@ public void accept(Packet packet) {
 
     @Override
     public OnJoinRegistrationOperation getPreJoinOperation() {
-        // pre-join operations are only sent by master member
```

Comment:
maybe keep this comment and add that specifically `EventService`'s pre-join op is also sent from joining member to master in `JoinRequest`?